html/contributeRoute.html

```diff
@@ -7,6 +7,29 @@
     <title>Document</title>
 </head>
 <body>
+    <form action="/my-handling-form-page" method="post">
+          <div>
+            <label for="contributer_id">User ID:</label>
```

Comment:
Is this going to be a unique ID or just the username? Otherwise I would just make it userName

Reply:
Also I think they should just be users in general as they can be both a user and contributer

Reply:
@rfpoulos I'm doneeeee

Reply:
@rfpoulos I'm done as well - looks like a great start!